ansible/deploy.yml

```diff
@@ -286,12 +344,19 @@
     - role: dashmate
     - role: mn_status_report
     - role: metricbeat
+  tags:
+    - full_deploy
+    - platform_update
```

Comment:
No need to do this more than once, maybe once in a blue moon when Pasta requires it... leave it as just full_deploy for now